DID.as

```diff
@@ -81,14 +86,14 @@ namespace DID {
             DID::drawString(leftPadded[i], vec2(leftOffset, 0)+diegeticCustomOffset.xy, diegeticCustomOffset.z + i*diegeticLineSpacing*-1);
             nvg::StrokeColor(lanes[i+4].color);
             DID::drawString(lanes[i+4].content, vec2(rightOffset, 0)+diegeticCustomOffset.xy, diegeticCustomOffset.z + i*diegeticLineSpacing*-1);
-        }    
+        }
     }
 
     DID::LaneConfig@[] lanes;
-    dictionary font;
+
 
     void Main() {
-        init();
+        OnSettingsChanged();
```

Comment:
for my sanity i prefer to keep all this stuff in a distinct init() called both at plugin startup and when things change that require reloading that kinda thing

Reply:
Moved things around, hopefully makes more sense now.